manifests/plugin_bands.pp

```diff
@@ -6,7 +6,15 @@
   $git_branch = "master",
 
   # INI settings below
-  $stage_agreement_deadline = undef,
+  $auction_start = "2016-02-21 11",
+  $band_panel_deadline = "2015-12-20",
+  $band_bio_deadline = "2016-01-03",
+  $band_agreement_deadline = "2016-01-17",
+  $band_w9_deadline = "2016-01-17",
+  $band_merch_deadline = "2016-01-31",
+  $band_charity_deadline = "2016-01-31",
+  $band_badge_deadline = "2016-02-07",
+  $stage_agreement_deadline = "2016-02-07",
```

Comment:
I'd prefer if we didn't have any fake data in here, I've been moving us away from that (despite still having a lot of that currently going on in the main uber config.pp file).

I think (not sure) that all this might still work OK if you set all these to undef.